pkgs/applications/misc/shoutrrr/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
```

Comment:
Consider removing the `rec` for the `let in` synatx, see here https://nix.dev/guides/best-practices.html.
```suggestion
let
  version = "0.8.0";
in
buildGoModule  {
```

Reply:
done